### ISSUE TITLE
Error in console on player login (latest dev build)

### ISSUE THREAD

**Issue by BaxAndrei** [craftaro/FabledSkyBlock#39]
```
[17:16:30] [Craft Scheduler Thread - 16 - FabledSkyBlock/WARN]: [FabledSkyBlock] Plugin FabledSkyBlock v3.0.5 generated an exception while executing task 34
java.util.ConcurrentModificationException: null
	at java.util.HashMap$HashIterator.nextNode(Unknown Source) ~[?:?]
	at java.util.HashMap$KeyIterator.next(Unknown Source) ~[?:?]
	at com.craftaro.skyblock.island.IslandManager.getVisitorsAtIsland(IslandManager.java:1192) ~[FabledSkyBlock.jar:?]
	at com.craftaro.skyblock.placeholder.PlaceholderProcessor.processPlaceholder(PlaceholderProcessor.java:293) ~[FabledSkyBlock.jar:?]
	at com.craftaro.skyblock.placeholder.hook.PlaceholderAPI.onPlaceholderRequest(PlaceholderAPI.java:35) ~[FabledSkyBlock.jar:?]
	at me.clip.placeholderapi.PlaceholderHook.onRequest(PlaceholderHook.java:32) ~[PlaceholderAPI.jar:?]
	at me.clip.placeholderapi.replacer.CharsReplacer.apply(CharsReplacer.java:119) ~[PlaceholderAPI.jar:?]
	at me.clip.placeholderapi.PlaceholderAPI.setPlaceholders(PlaceholderAPI.java:71) ~[PlaceholderAPI.jar:?]
	at me.clip.placeholderapi.PlaceholderAPI.setPlaceholders(PlaceholderAPI.java:99) ~[PlaceholderAPI.jar:?]
	at com.craftaro.skyblock.placeholder.PlaceholderManager.parsePlaceholders(PlaceholderManager.java:36) ~[FabledSkyBlock.jar:?]
	at com.craftaro.skyblock.scoreboard.Holder.update(Holder.java:34) ~[FabledSkyBlock.jar:?]
	at com.craftaro.skyblock.scoreboard.Driver.run(Driver.java:91) ~[FabledSkyBlock.jar:?]
	at org.bukkit.craftbukkit.v1_20_R3.scheduler.CraftTask.run(CraftTask.java:101) ~[paper-1.20.4.jar:git-Paper-464]
	at org.bukkit.craftbukkit.v1_20_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57) ~[paper-1.20.4.jar:git-Paper-464]
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22) ~[paper-1.20.4.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) ~[?:?]
	at java.lang.Thread.run(Unknown Source) ~[?:?]
```

**Comment by SpraxDev** [comment]
Thanks for your report - This should be fixed in v3.0.6 released just now

Please reopen the issue with a new error log if the error occurs again

**Comment by BaxAndrei** [comment]
Everything is fine! Thank you!
